src/VirtualDataRepeater.js

```diff
@@ -70,18 +70,7 @@ module.exports = kind({
 
 		this.stabilizeExtent();
 
-		var refresh = this.bindSafely(function() {
-			this.doIt();
-		});
-
-		// refresh is used as the event handler for
-		// collection resets so checking for truthy isn't
-		// enough. it must be true.
-		if(immediate === true) {
-			refresh();
-		} else {
-			this.startJob('refreshing', refresh, 16);
-		}
+		this.doIt();
```

Comment:
This change makes the `immediate` parameter unused and deprecated. Also, does it actually need to refresh when hidden? If lots of changes are happening in succession this would waste a lot of time. What if refreshing were handled much like the new reflow-when-not-showing system forcing it to update when showing changes? I think this is another reason why it was async to begin with in DataList because if lots of sequential updates were happening to the collection we didn't need to try and update the list over and over for each change...